tests/jest/unit/app/modules/import-export/assets/js/admin.test.js

```diff
@@ -0,0 +1,121 @@
+describe( 'Import Export Admin Test', () => {
+	window.elementorAppConfig = {
+		'import-export': {
+			lastImportedSession: {
+				kit_title: 'Kit Title',
+			},
+		},
+	};
+
+	require( 'elementor/app/modules/import-export/assets/js/admin.js' );
```

Comment:
use regular import

Reply:
the admin file does not export anything and it doesn't need to